src/pages/AppLogin.vue

```diff
@@ -71,101 +83,85 @@ onMounted(async () => {
-            >
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('google')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/google-sso.svg"
-                  />
-                  <span class="body-1">Google</span>
-                </div>
-              </v-card-button>
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('github')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/github-sso.svg"
-                  />
-                  <span class="body-1">Github</span>
-                </div>
-              </v-card-button>
-            </div>
-            <div
-              class="flex wrap"
-              style="gap: 1em; justify-content: space-between; margin-top: 2em"
-            >
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('twitch')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/twitch-sso.svg"
-                  />
-                  <span class="body-1">Twitch</span>
-                </div>
-              </v-card-button>
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('discord')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/discord-sso.svg"
-                  />
-                  <span class="body-1">Discord</span>
-                </div>
-              </v-card-button>
-            </div>
+      <LandingDescriptor />
+      <section class="signin-section">
+        <div class="signin-container">
+          <img
+            src="@/assets/logo.svg"
+            alt="Arcana Logo"
+            style="margin: auto; margin-bottom: 2em"
+            class="laptop-remove"
+          />
+          <div>
+            <h1>Welcome</h1>
+            <h5 class="login-description">
+              We’ll email you a magic link for a password-free sign in.
+            </h5>
+          </div>
+          <div class="passwordless-container flex column flex-center">
+            <VTextField
+              v-model.trim="email"
+              label="Email"
+              class="passwordless-email"
+              message="Invalid Email"
+              :message-type="hasValidEmail ? '' : 'error'"
+              @update:model-value="validateEmail"
```

Comment:
We don't want show the validation error on every input event as it is disruptive during the course of typing in the email address. Can we validate [on blur](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event) instead?

https://user-images.githubusercontent.com/11095038/185116961-a0640992-90c1-4868-819f-e690febdf3da.mov

Reply:
Removed the error message for now and kept the button disabled until email is valid. Displaying message on `blur` becomes a bit tricky
Resolved at https://github.com/arcana-network/developer-dashboard/pull/121/commits/7b00c57f2704f07bd58178b721d4f56fff015592